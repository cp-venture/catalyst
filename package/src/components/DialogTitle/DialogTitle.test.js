import React from "react";
import { cleanup, render } from "../../tests/index.js";
import DialogTitle from "./DialogTitle";

afterEach(cleanup);

test("basic snapshot - only default props", () => {
  const { asFragment } = render(<DialogTitle>Archive 24 products?</DialogTitle>);
  expect(asFragment()).toMatchSnapshot();
});
