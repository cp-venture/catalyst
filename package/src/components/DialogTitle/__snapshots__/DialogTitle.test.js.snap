// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic snapshot 1`] = `
<div
  className="MuiDialogTitle-root"
>
  <h2
    className="MuiTypography-root MuiTypography-h4"
  >
    Archive 24 products?
  </h2>
</div>
`;
